src/components/Button/index.tsx

```diff
@@ -52,6 +53,7 @@ const Button = (props: TButtonPropTypes): JSX.Element => {
       )}
       onClick={onClick}
       form={formId}
+      autoFocus={autoFocus}
```

Comment:
suggest to extend Button types from generic button type

export interface TButtonPropTypes
  extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'type'> {}
  
  With this way you can not describe `autoFocus` type in types

Reply:
I use Omit so that Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'type'> is used to create a type that includes all properties of ButtonHTMLAttributes<HTMLButtonElement> except for the 'type' property. This is done to avoid conflicts between the type property in the default button attributes and the type property in your custom TButtonPropTypes interface.